src/helm/proxy/tokenizers/lit_gpt_tokenizer.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Dict
+from pathlib import Path
+
+import torch
+
+from helm.common.cache import CacheConfig
+from helm.common.optional_dependencies import handle_module_not_found_error
+from .caching_tokenizer import CachingTokenizer
+
+try:
+    from lit_gpt import Tokenizer as InternalTokenizer
+    from lit_gpt.utils import check_valid_checkpoint_dir
+except ModuleNotFoundError as e:
+    handle_module_not_found_error(e)
+
+
+class LitGPTTokenizer(CachingTokenizer):
```

Comment:
Note that the Lit-GPT tokenizer was a singleton before but not is not a singleton here.